### ISSUE TITLE
Add to calendar for reminders

### ISSUE THREAD

**Issue by OlivierJM** [OlivierJM/powercut#11]
**Describe the solution you'd like**
It would be nice to a way for users to save a schedule as a calendar event that maybe spans for a week. 

**Describe alternatives you've considered**
Alternative is sending web push notifications

**Additional context**

source: https://www.reddit.com/r/Zambia/comments/1bhkgim/comment/kvel62i/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by Mwimwii** [comment]
After a bit of thought using my marketing/business hat, this could be a double edged sword.

On one side, it could drive people away from frequently using the application.
On the other, we'll be getting google sign ins, which is great but what will we be using them for.

I propose that we probably go for a **"save location"** feature where we just save the users home/work/etc so that they can instantly just be met with the schedule. I believe the [Recent search feature](#18 ) is a good step once its merged to master

We can then be able to use the [Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) to send reminders instead of relying on Google's Calender API

**Comment by OlivierJM** [comment]
I totally agree with you on this @Mwimwii  Let's put it on hold for now until we can get to notifications which also shouldn't be a priority.
